numerics_paper/version2/numerical_chj.tex

```diff
@@ -100,6 +100,13 @@ \subsection{Numerical shock instabilities}
 Most notably, dissipative Riemann solvers like HLLE and Rusanov suppress the carbuncle instability \cite{quirk1997contribution}.
 For a recent review of numerical shock instability and work to alleviate it,
 we refer to \cite[Section 2.5]{simonnumerical} and \cite{zaide2012numerical}.
+{\color{OliveGreen}
+  Another important shock anomaly is the presence of a spike in the momentum downstream 
```

Comment:
I thought the spike was inside the shock, not downstream?

Reply:
In my opinion, it's enough to discuss the spike later in our experiments, and not in the introduction, since it is not part of the purpose of our paper.